drivers/oled/oled_driver.c

```diff
@@ -71,6 +71,9 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #define PRE_CHARGE_PERIOD 0xD9
 #define VCOM_DETECT 0xDB
 
+// Advance Graphic Commands
+#define FADE_BLINK 0x23
```

Comment:
I can't find any reference to this command in the spec sheet. Able to link me to where you found this info?

Reply:
Whelp, just found it right after posting it.

Reply:
> Whelp, just found it right after posting it.

Naturally. :D